.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+	"presets": [
+		"es2015",
+		"stage-1",
```

Comment:
Ultimately it's up to you which level of ECMA proposals you want to incorporate. Keep in mind that stage 1 includes some more unstable features like [decorators](http://tc39.github.io/proposal-decorators/) which I consider to be "riskier" (at least as far as potentially being dropped from specification consideration altogether).